_entries/02-01 create cluster.md

```diff
@@ -5,238 +5,6 @@ title: Create cluster
-az provider register -n Microsoft.Solutions --wait
-az provider register -n Microsoft.Network --wait
-az provider register -n Microsoft.KeyVault --wait
-az provider register -n Microsoft.ContainerService --wait
-```
-
-{% endcollapsible %}
-
-#### Create the cluster
-
-{% collapsible %}
-
-1. Create a resource group to hold the resources, for example in East US.
-
-    ```bash
-    az group create --name aroworkshop --location eastus
-    ```
-
-1. You’re now ready to create a cluster. The following will create the cluster in the specified Azure AD tenant, specify the Azure AD app object and secret to use as a security principal, and the security group that contains the members that have admin access to the cluster.
-
-    Make sure to replace the placeholders for `<app id>`, `<secret>`, `<tenant id>` and `<group id>` with the values you saved from the earlier steps. Also replace `<cluster name>` with a unique name for your cluster.
-
-    ```bash
-    az openshift create \
-        --resource-group aroworkshop \
-        --location eastus \
-        --name <cluster name> \
-        --aad-client-app-id <app id> \
-        --aad-client-app-secret <secret> \
-        --aad-tenant-id <tenant id> \
-        --customer-admin-group-id <group id>
-    ```
-
-{% endcollapsible %}
-
-To be able to login to the cluster, you’ll need to update the app registration you created with the sign in URL of your cluster. This will enable Azure Active Directory authentication to properly redirect back to your cluster's web console after successful authentication.
-
-#### Update your app registration redirect URI
-
-{% collapsible %}
-
-1. Get the sign in URL for your cluster. Replace `<cluster name>` with the name of your cluster.
-
-    ```bash
-    az openshift show -n <cluster name> -g aroworkshop --query "publicHostname" -o tsv
-    ```
-
-    You should get back something like `openshift.xxxxxxxxxxxxxxxxxxxx.eastus.azmosa.io`. The sign in URL for your cluster will be https:// followed by the publicHostName value, for example, <https://openshift.xxxxxxxxxxxxxxxxxxxx.eastus.azmosa.io>.
-
-1. Now that you have the sign in URL for the cluster, set the app registration
-redirect UI. Open the [App registrations](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredAppsPreview) blade.
-
-1. Click on your app registration object.
-
-1. Click on **Add a redirect URI**.
-
-1.  Ensure that **Type** is **Web** and set the **Redirect URI** using the following pattern: **https://<public host name>/oauth2callback/Azure%20AD**. For example: <https://openshift.xxxxxxxxxxxxxxxxxxxx.eastus.azmosa.io/oauth2callback/Azure%20AD>
-
-    ![Grant Admin consent](../media/redirect-uri.png)
-
-1. Click Save.
-
-{% endcollapsible %}
\ No newline at end of file
+We will now create our own ARO cluster, following the steps of the documentation [to create an ARO cluster](https://docs.microsoft.com/en-us/azure/openshift/tutorial-create-cluster). 
```

Comment:
Perhaps it is better to bring the instructions from the tutorial here. Past feedback has been people find clicking outside the workshop to follow instructions confusing.

Reply:
I could. In fact, I did inject the documantion 5 days ago in the first PR and I edited because...the documentation was faulty.
I opened a ticket they fixed it (https://github.com/MicrosoftDocs/azure-docs/issues/53315). If you feel that documentation is up to date, I will inject it (but we may have to keep them sync)

Reply:
Yes, let's pull the instructions here :)

Reply:
sir, yes sir :)

Reply:
Done